pkg/prometheus/promcfg_test.go

```diff
@@ -1244,6 +1243,69 @@ func TestAlertmanagerEnableHttp2(t *testing.T) {
 	}
 }
 
+func TestAlertmanagerAlertRelabelConfigs(t *testing.T) {
+	for _, tc := range []struct {
+		name    string
+		version string
+		golden  string
+	}{
+		{
+			name:    "Invalid Prometheus Version",
+			version: "2.40.1",
+			golden:  "AlertmangerAlertRelabel_Invalid_Version.golden",
+		},
+		{
+			name:    "Valid Prometheus Version",
+			version: "2.51.0",
+			golden:  "AlertmangerAlertRelabel_Valid_Version.golden",
+		},
+	} {
+		t.Run(fmt.Sprintf("AlertmanagerAlertRelabelConfigs Case: %s Version: %s", tc.name, tc.version), func(t *testing.T) {
```

Comment:
```suggestion
		t.Run(tc.name, func(t *testing.T) {
```

Just tried the test myself and it took almost a full line 😵. The the test function name, appended by tc.name looks good enough :)